test/src/org/labkey/test/tests/panoramapremium/TargetedMSSampleManagerIntegrationTest.java

```diff
@@ -114,10 +115,7 @@ public void testSampleTypeNavigation()
     private void createSampleType(String sampleName)
     {
         projectMenu().navigateToFolder(getProjectName(), Sample_Manager_Subfolder);
-        SampleTypeHelper sampleTypeHelper = new SampleTypeHelper(this);
-        SampleTypeDefinition sampleTypeDefinition = new SampleTypeDefinition(sampleName);
-        sampleTypeHelper.createSampleType(sampleTypeDefinition);
-        sampleTypeHelper.goToSampleType(sampleName);
+        SampleTypeAPIHelper.createEmptySampleType(getProjectName(), new SampleTypeDefinition(sampleName));
```

Comment:
This doesn't look like it's creating the sample type in the correct location
```suggestion
        SampleTypeAPIHelper.createEmptySampleType(getProjectName() + Sample_Manager_Subfolder, new SampleTypeDefinition(sampleName));
```